jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-cluster-jbpm-schema.sql

```diff
@@ -0,0 +1,853 @@
+        name varchar(255),
+        attachment_size integer,
+        attachedBy_id varchar(255),
+        TaskData_Attachments_Id bigint,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table AuditTaskImpl (
+        id bigint not null auto_increment,
+        activationTime datetime(6),
+        actualOwner varchar(255),
+        createdBy varchar(255),
+        createdOn datetime(6),
+        deploymentId varchar(255),
+        description varchar(255),
+        dueDate datetime(6),
+        name varchar(255),
+        parentId bigint not null,
+        priority integer not null,
+        processId varchar(255),
+        processInstanceId bigint not null,
+        processSessionId bigint not null,
+        status varchar(255),
+        taskId bigint,
+        workItemId bigint,
+        lastModificationDate datetime(6),
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table BAMTaskSummary (
+        pk bigint not null auto_increment,
+        createdDate datetime(6),
+        duration bigint,
+        endDate datetime(6),
+        processInstanceId bigint not null,
+        startDate datetime(6),
+        status varchar(255),
+        taskId bigint not null,
+        taskName varchar(255),
+        userId varchar(255),
+        OPTLOCK integer,
+        primary key (pk)
+    ) ENGINE=InnoDB;
+
+    create table BooleanExpression (
+        id bigint not null auto_increment,
+        expression longtext,
+        type varchar(255),
+        Escalation_Constraints_Id bigint,
+        primary key (id)
+    ) ENGINE=InnoDB;
+    
+    create table CaseIdInfo (
+        id bigint not null auto_increment,
+        caseIdPrefix varchar(255),
+        currentValue bigint,
+        primary key (id)
+    ) ENGINE=InnoDB;
+    
+    create table CaseFileDataLog (
+        id bigint not null auto_increment,
+        caseDefId varchar(255),
+        caseId varchar(255),
+        itemName varchar(255),
+        itemType varchar(255),
+        itemValue varchar(255),
+        lastModified datetime(6),
+        lastModifiedBy varchar(255),
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table CaseRoleAssignmentLog (
+        id bigint not null auto_increment,
+        caseId varchar(255),
+        entityId varchar(255),
+        processInstanceId bigint not null,
+        roleName varchar(255),
+        type integer not null,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table Content (
+        id bigint not null auto_increment,
+        content longblob,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table ContextMappingInfo (
+        mappingId bigint not null auto_increment,
+        CONTEXT_ID varchar(255) not null,
+        KSESSION_ID bigint not null,
+        OWNER_ID varchar(255),
+        OPTLOCK integer,
+        primary key (mappingId)
+    ) ENGINE=InnoDB;
+
+    create table TimerMappingInfo (
+    	id bigint generated not null auto_increment, 
```

Comment:
`generated` is not required here. Please remove
```suggestion
    	id bigint not null auto_increment, 
```

Reply:
done